certified-connectors/LatinShare SHP Permissions/apiDefinition.swagger.json

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
@NicolasValenzuela This swagger file is not up to date, can you please update it with the latest version.

Reply:
Okay, it's up to date with the latest version. I'll let you know when it's ready